libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -1646,22 +1646,11 @@ class MapboxRouteLineApi(
 
         val primaryRouteCasingExpressionProducer =
             RouteLineExpressionProvider {
-                if (routeLineOptions.styleInactiveRouteLegsIndependently) {
```

Comment:
Why does the base layer still need an independent code path for inactive leg styling and casing layer doesn't?

Reply:
We don't have an "inactive leg casing colour" option. It was using travelled colour instead. But I now see that we use travelled casing colour in `updateTraveledRouteLine` for inactive legs. Is this expected?

Reply:
Could you help here @cafesilencio?

Reply:
Let me revert this change and make it a separate PR. The width problems are fixed by MapboxRouteLineView changes.

Reply:
Here's the video of how the colours work on main @cafesilencio:

https://user-images.githubusercontent.com/32109537/235144208-ec3369c2-60fa-46c6-8969-690220bb2f28.mp4